README.md

```diff
@@ -4,6 +4,9 @@
 
 The World Health Assembly is the decision-making body of WHO. It is attended by delegations from all WHO Member States and focuses on a specific health agenda prepared by the Executive Board. The main functions of the World Health Assembly are to determine the policies of the Organization, appoint the Director-General, supervise financial policies, and review and approve the proposed programme budget. The Health Assembly is held annually in Geneva, Switzerland.
 
+
+![screenshot](images/Screenshot2.png)
+
```

Comment:
- [ ] Like the last code reviewer said, kindly add an acknowledgment section in the readme file that should include a design link. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)